script/deployers/MumbaiMangroveFullTestnetDeployer.s.sol

```diff
@@ -51,31 +64,41 @@ contract MumbaiMangroveFullTestnetDeployer is Deployer {
     IERC20 dai = IERC20(fork.get("DAI"));
     IERC20 usdc = IERC20(fork.get("USDC"));
     IERC20 weth = IERC20(fork.get("WETH"));
+
+    uint[] memory prices = priceOracle.getAssetsPrices(dynamic([address(dai), address(usdc), address(weth)]));
+    maticPrice = priceOracle.getAssetPrice(fork.get("WMATIC"));
+
+    // 1 token_i = (prices[i] / 10**8) USD
+    // 1 USD = (10**8 / maticPrice) Matic
+    // 1 token_i = (prices[i] * 10**9 / maticPrice) gwei of Matic
     new ActivateMarket().innerRun({
       mgv: mgv,
+      gasprice: 140,
```

Comment:
Why does this not use the same gasprice as Mangrove?

Reply:
I tried to explain in the description of the PR above. The idea is to be able to set a realistic density (the one we expect to see on mainnet) while keeping a low gasprice in order to avoid requiring tester to provide too many matics.

Reply:
Ok. This explanation should be in this script as it's non-obvious and will soon be forgotten :-)

Reply:
I'll push something